packages/js/client-config-builder/src/bundles/getDefaultConfig.ts

```diff
@@ -46,6 +47,7 @@ export const defaultInterfaces = {
   uriResolver: "wrap://ens/uri-resolver.core.polywrap.eth",
   concurrent: "wrap://ens/goerli/interface.concurrent.wrappers.eth",
   logger: "wrap://ens/wrappers.polywrap.eth:logger@1.0.0",
+  ethereumProvider: "wrap://ens/iprovider.polywrap.eth",
```

Comment:
Can we have a better name here?
Ex: `provider.ethereum.polywrap.eth` ?

Reply:
The purpose is to clarify that it refers to the provider interface, not to an implementation. I actually don't like the ambiguous naming used elsewhere.

Reply:
It's ambiguous since it doesn't mention it's ethereum provider interface. Maybe
`iprovider.ethereum.polywrap.eth` or `interface.provider.ethereum.polywrap.eth`?

Reply:
Okay, I'll change it.

A name prefixed with `i` usually means interface (e.g. `IUriWrapper`, `IUriPackage`, `IUriResolver`). So I thought it made sense here. But I don't want it to be unclear to other people.

Reply:
i prefix isn't the problem. not mentioning the `ethereum` in uri is ambiguous.  There can be many kinds of providers EX: IPFS Provider, Tezos Provider, Near Provider, etc

Reply:
ohhhh I see what you're saying

Reply:
We never prefixed the interface with `i` in the URI before (maybe we should?) 
Ex: we assumed ethereum interface to be deployed on `ethereum.polywrap.eth`, ipfs on `ipfs.polywrap.eth`. it was alright since we were redirecting these interface uris to plugins but we should make distinction more clear.

We are still doing the same thing with the logging/logger :https://github.com/polywrap/logging/blob/master/aggregator/resources/README.md

Where logger interface is deployed on the `ens/wrappers.polywrap.eth:logger@1.0.0`. This uri doesn't convey that it's an interface. 

While I always have been adding interface subdomain to actual wrapper domain to show that uri is an interface (Ex: `interface.price.resolver.defiwrapper.eth`)  and an implementation at: `coingecko.price.resolver.defiwrapper.eth`

The question is whether we should explicitly add prefixes to show the uri is interface or a wrapper?

Reply:
Yeah, I see. You're 100% right. 

Right now we are publishing everything to `ens/wrappers.polywrap.eth`. Maybe we should have `ens/interfaces.polywrap.eth` as well?

Reply:
I updated with the uri `ens/wrappers.polywrap.eth:ethereum-provider@1.0.0`

Reply:
Can I mark this resolved?